src/engraving/libmscore/slur.cpp

```diff
@@ -765,11 +912,16 @@ void Slur::slurPos(SlurPos* sp)
             pt.rx() = hw2 * 0.5;
         }
         // don't allow overlap with beam
-        qreal yadj = ec->beam() ? 0.75 : -0.25;
+        qreal yadj;
+        if (ec->beam() && ec->beam()->elements().first() != ec) {
```

Comment:
Doesn't build, at least not with MSVC:
"first()" is not a member of 'stdvector<Ms::ChordRest *, std::allocator<Ms::ChordRest *>>'

Reply:
Should it be `        if (ec->beam() && ec->beam()->elements().front() != ec) {` instead?

Reply:
That actually dows work, see #10851

Reply:
After a bit of investigation, commit https://github.com/musescore/MuseScore/commit/94d7ee71f85e7fddc342bfe4208a499af5476133 seems to have changed QVector (which does have a `first` method) with std::vector (which does not). Your suggestion is correct!